Sources/SwiftInspectorAnalyzers/PropertyAnalyzer.swift

```diff
@@ -37,15 +37,11 @@ public final class PropertyAnalyzer: Analyzer {
 
   /// Finds property information for the provided type in the Swift file
   /// - Parameter fileURL: The fileURL where the Swift file is located
-  public func analyze(fileURL: URL) throws -> TypeProperties? {
+  public func analyze(fileURL: URL) throws -> Set<PropertyData>? {
```

Comment:
The only additional information we had in `TypeProperties` was the type name. But we already know that since the `TypeSyntaxVisitor` is instantiated with a type name.